Sources/Commands/SwiftTool.swift

```diff
@@ -455,7 +455,7 @@ private func findPackageRoot() -> AbsolutePath? {
     // FIXME: It would be nice to move this to a generalized method which takes path and predicate and
     // finds the lowest path for which the predicate is true.
     var root = currentWorkingDirectory
-    while !isFile(root.appending(component: Manifest.filename)) {
+    while !isFile(Manifest.versionSpecificManifest(path: root)) {
```

Comment:
We should require that `Package.swift` be present at top level. So please revert this line.